components/rewards/RewardsHome.js

```diff
@@ -1,65 +1,68 @@
 import React from 'react';
-import { ScrollView, Text, View } from 'react-native';
-import { Body, Overline } from '../BaseComponents';
-import { ProgressBar, Colors } from 'react-native-paper';
-import RewardsCard from '../../components/rewards/RewardsCard';
-import { ContentContainer, WaluigiContainer } from '../../styled/rewards';
+import { View, ScrollView } from 'react-native';
+import { Body, Overline, Title } from '../BaseComponents';
+import {
+  ProgressBarContainer,
+  RewardCardContainer
+} from '../../styled/rewards';
+import { ProgressBar } from 'react-native-paper';
+import RewardTab from './RewardTab';
+import Colors from '../../assets/Colors';
 
 /**
  * @prop
  * */
 
-function createList(N) {
+function createPairs(N) {
   let foo = [];
-
-  for (var i = 1; i <= N; i++) {
+  for (var i = 1; i <= N / 2; i++) {
     foo.push(i);
   }
   return foo;
 }
 
+function createSingle(N) {
+  let foo = [];
+  if (N % 2 == 1) {
+    foo.push(1);
+  }
+  return foo;
+}
+// TODO @kennethlien to fix infinite scroll
 function RewardsHome({ transactions, user }) {
   return (
     <View>
       <ScrollView style={{ marginTop: 45 }}>
-        <WaluigiContainer>
-          <Overline>REWARD PROGRESS</Overline>
-          <Text style={{ fontSize: 10 }}> </Text>
-          <Text
-            style={{
-              fontSize: 30
-            }}>
+        <ProgressBarContainer>
```

Comment:
This naming is confusing -- it would seem like a container for the progress bar alone. Can you change this to RewardProgressContainer or something?

Reply:
Actually I just realized the Available Rewards overline is also in this. But also see my other comment about how you can also put the cards into here and rid of the extra 5% left/right margin